src/test/scala/zio/sqs/producer/ProducerSpec.scala

```diff
@@ -178,13 +178,13 @@ object ProducerSpec extends DefaultRunnableSpec {
           assert(innerReqEntries.head.messageAttributes.getOrElse(Map.empty).size)(equalTo(1)) &&
           assert(innerReqEntries.head.messageAttributes.getOrElse(Map.empty).contains("Name"))(isTrue) &&
           assert(innerReqEntries.head.messageAttributes.getOrElse(Map.empty)("Name"))(equalTo(attr)) &&
-          assert(innerReqEntries.head.messageGroupId)(isSome(equalTo("g1"))) &&
-          assert(innerReqEntries.head.messageDeduplicationId)(isSome(equalTo("d1"))) &&
+          assert(innerReqEntries.head.messageGroupId.toOption)(isSome(equalTo("g1"))) &&
```

Comment:
What's the new type for stuff that need `toOption`?

Reply:
`innerReqEntries.head.messageGroupId` has type `zio.prelude.data.Optional[String]`, but `isSome` has type `Assertion[Option[String]]`:

```scala
package zio.aws.sqs.model
import zio.ZIO
import zio.aws.core.{AwsError, BuilderHelper}
import zio.prelude.data.Optional
import zio.aws.sqs.model.primitives.Integer
import scala.jdk.CollectionConverters._
final case class SendMessageBatchRequestEntry(
    id: String,
    messageBody: String,
    delaySeconds: Optional[Integer] = Optional.Absent,
    messageAttributes: Optional[
      Map[String, zio.aws.sqs.model.MessageAttributeValue]
    ] = Optional.Absent,
    messageSystemAttributes: Optional[Map[
      zio.aws.sqs.model.MessageSystemAttributeNameForSends,
      zio.aws.sqs.model.MessageSystemAttributeValue
    ]] = Optional.Absent,
    messageDeduplicationId: Optional[String] = Optional.Absent,
    messageGroupId: Optional[String] = Optional.Absent
) {
```

We either need an `Assertion[Optional[String]]`, or a conversion via `.toOption`.